test/integration/apiserver/apiserver_partition_test.go

```diff
@@ -43,6 +43,7 @@ import (
 const (
 	tenant1 = "tenant1"
 	tenant2 = "tenant2"
+	tenant3 = "tenant3"
```

Comment:
I would like to see the api server IP address defined here and referred in the test cases, instead of hard-coded in the test case code.